app/models/user.rb

```diff
@@ -1,9 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] I would recommend enabling the `:confirmable` key so that the application asks for confirmation of email and devise will simply handle.

Reply:
addressed this